exchanges/ticker/ticker.go

```diff
@@ -129,6 +131,22 @@ func ProcessTicker(p *Price) error {
 		return fmt.Errorf("%s %s", p.ExchangeName, errPairNotSet)
 	}
 
+	if p.Bid != 0 && p.Ask != 0 && p.AssetType != asset.MarginFunding { // Margin funding books can be crossed see Bitfinex.
```

Comment:
This looks like another iffy if where I think you should be looking to make the specific scenario of bitfinex skip and all others go into this